composer.json

```diff
@@ -13,6 +13,7 @@
     ],
     "require": {
         "php": "^7.2 || ^8.0",
+        "ext-mongodb": "^1.0",
```

Comment:
I don't think it's actually needed here. This package depends on `doctrine/mongodb-odm ^2.0` which requires `ext-mongodb ^1.5` already so this requirement is superfluous.

Reply:
Since we use `MongoDB\BSON\fromPHP` and `MongoDB\BSON\toCanonicalExtendedJSON`, I think we should define this dependency explicitly.
But, you're right, we can set it to `^1.5`.

Reply:
Picking the lowest version that contains the feature set you want makes sense. Adding the constraint is necessary, but 1.0 would've done. I'm not going to ask you to change it back though